### PR TITLE
Update Portuguese Translation

### PR DESCRIPTION
**Pull request by altiereslima**

### CODE REVIEW

**Comment by ihhub** [comment]
Hi @altiereslima , please check logs of failed stages. You can find the description of the problem with your changes

File: files/lang/pt.po [hunk 1]
```diff
@@ -132,7 +132,9 @@ msgstr ""
 msgid ""
 "CAMPAIGN\n"
 "GAME"
-msgstr "CAMPANHA"
```
Comment (ihhub) [inline files/lang/pt.po:135]: Could you please explain this change? Was the original word longer than the button?

**Review by ihhub — COMMENTED** [review]
Hi @altiereslima , I left one question here. Could you please check it?

File: files/lang/pt.po (hunk 1 above)
Comment (altiereslima) [reply files/lang/pt.po:135]: image
Exactly, it doesn’t fit on the button.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@altiereslima , thank you so much for this update!

**Pull request merged**